src/DIRAC/WorkloadManagementSystem/ConfigTemplate.cfg

```
@@ -19,6 +19,22 @@ Services
     }
   }
   ##END
+  TornadoPilotLogging
+  {
+    Protocol = https
+    Authorization
+    {
+      Default = authenticated
+      sendMessage = "Operator"
+      sendMessage += "Pilot"
+      sendMessage += "GenericPilot"
```

```suggestion
      sendMessage += "GenericPilot"
```
I think the "private pilot" is not needed here.